### DISCUSSION TITLE
RPi Model B+ no pi-hole responses

### DISCUSSION THREAD

**Discussion by rapierbit** [pi-hole/pi-hole#4608] [Bugs]
Not sure whether to post this here or on the Discord server. The only category I'm allowed is Bugs but I don't know that this is a bug.

Briefly, I have a Model B+, and I read that others are using the B+ for pi-hole. I've tried different OS installations, but currently  the B+ is running Buster Jan 28 2022 (lite). 

I've done the install you provide (curl -sSL https://install.pi-hole.net | bash) without incident and it seems to install just fine; however, pi-hole is non-responsive. I'm using a static IP on a wired network (Ubiquiti UDM); the installer detected this and I selected all default prompts. The IP is 192.168.69.69, and this is the only IP address on this network, as configured in my UDM.

I can ping the B+ just fine, but when I point any queries at it, I get nada (see below). What's my next best troubleshooting step (go to Discord?). One thing I note from the admin interface is that the although the load looks ok (below 0.5 typically), the RAM utilization is only 14.7%. Based on other forum posts I've read for the module B+, this doesn't seem correct (very low). Screenshot attached.

~ dirkdark$ ping 192.168.69.69
PING 192.168.69.69 (192.168.69.69): 56 data bytes
64 bytes from 192.168.69.69: icmp_seq=0 ttl=63 time=0.901 ms
64 bytes from 192.168.69.69: icmp_seq=1 ttl=63 time=0.901 ms
64 bytes from 192.168.69.69: icmp_seq=2 ttl=63 time=0.877 ms

~ dirkdark$ dig www.microsoft.com
; <<>> DiG 9.10.6 <<>> www.microsoft.com
;; global options: +cmd
;; Got answer:
;; ANSWER SECTION:
www.microsoft.com.	969	IN	CNAME	www.microsoft.com-c-3.edgekey.net.
www.microsoft.com-c-3.edgekey.net. 249 IN CNAME	www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net.
www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net. 249 IN CNAME e13678.dscb.akamaiedge.net.
e13678.dscb.akamaiedge.net. 5	IN	A	23.50.125.163

~ dirkdark$ dig @192.168.69.69 www.microsoft.com
; <<>> DiG 9.10.6 <<>> @192.168.69.69 www.microsoft.com
; (1 server found)
;; global options: +cmd
;; connection timed out; no servers could be reached

![14 7 memory](https://user-images.githubusercontent.com/38741158/153729544-556acf13-f5bd-4821-8194-93d5deeeb5ba.png)

**Comment by yubiuser** [comment]
You have two options: open a bug (https://github.com/pi-hole/pi-hole/issues) or (preferred) ask for help on discourse https://discourse.pi-hole.net/c/bugs-problems-issues/11

**Reply by rapierbit** [reply]
> You have two options: open a bug (https://github.com/pi-hole/pi-hole/issues) or (preferred) ask for help on discourse https://discourse.pi-hole.net/c/bugs-problems-issues/11

Thanks, done, sorry for the noise.
